---
title: '&#39;Catch&#39; blok hiçbir zaman ulaşıldı, çünkü &#39; &lt;Ad1&gt; &#39; devraldığı &#39; &lt;ad2&gt;&#39;'
ms.date: 07/20/2015
f1_keywords:
- BC42029
- vbc42029
helpviewer_keywords:
- BC42029
ms.assetid: 4bc84a84-ec7e-49db-8794-4fb71d36d5c0
ms.openlocfilehash: d97bff73092fe1b92d2596961b0ce93c7a585bc3
ms.sourcegitcommit: 3d5d33f384eeba41b2dff79d096f47ccc8d8f03d
ms.translationtype: MT
ms.contentlocale: tr-TR
ms.lasthandoff: 05/04/2018
ms.locfileid: "33633207"
---
# <a name="39catch39-block-never-reached-because-39ltname1gt39-inherits-from-39ltname2gt39"></a>&#39;Catch&#39; blok hiçbir zaman ulaşıldı, çünkü &#39; &lt;Ad1&gt; &#39; devraldığı &#39; &lt;ad2&gt;&#39;
A `Catch` kodu blokta hiç ulaşılamadı.  
  
 Varsayılan olarak, bu iletiyi bir uyarıdır. Uyarıları gizleme veya uyarıları hata olarak davranma daha fazla bilgi için bkz: [yapılandırma uyarılarını Visual Basic'te](/visualstudio/ide/configuring-warnings-in-visual-basic).  
  
 **Hata Kimliği:** BC42029  
  
## <a name="to-correct-this-error"></a>Bu hatayı düzeltmek için  
  
-   Yabancı kaldırmak `Catch` bloğu.  
  
## <a name="see-also"></a>Ayrıca Bkz.  
 [Nasıl yapılır: Test kodu ile bir Try... Visual Basic'te catch bloğu](http://msdn.microsoft.com/library/8368e205-ed73-4185-a247-af84fb4fafa9)  
 [Try...Catch...Finally Deyimi](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
